src/features/locators/types/locator.types.ts

```diff
@@ -95,7 +95,8 @@ export interface Locator extends PredictedEntity {
   active?: boolean;
   isCustomName?: boolean;
   isCustomLocator?: boolean;
-  locatorType?: LocatorType;
+  annotationType: AnnotationType;
+  locatorType: LocatorType;
```

Comment:
In fact, locator doesn't have this attribute until user set manually it in Edit Form. Changing it to requeried will brake the logic in several places

Reply:
Rolled back